integrated_channels/sap_success_factors/exporters/learner_data.py

```diff
@@ -25,6 +25,8 @@ class SapSuccessFactorsLearnerExporter(LearnerExporter):
     Class to provide a SAPSF learner data transmission audit prepared for serialization.
     """
 
+    INCLUDE_GRADE_FOR_COMPLETION_AUDIT_CHECK = False
```

Comment:
Could I get a TLDR on how this works? Will it override the mixin passed to the base exporter?